src/commands/sus.ts

```diff
@@ -95,15 +96,27 @@ export class SusCommand extends Command {
                 return prev;
             }, [] as Array<ReactionInfo>);
 
-        const isImposter = Math.random() < 0.5;
+        const skipVotes = msg.reactions.resolve(Emojis.SkipVote.snowflake).count;
+        let reason: string;
+        if (skipVotes >= mostVotes[0].count) {
+            reason = "Skipped.";
+        } else if (mostVotes.length > 0) {
```

Comment:
Replace with:
`} else if (mostVotes.length > 1) {`

Since exactly 1 reaction with 1 vote is what we are looking for to eject someone.